File: Flask/app/api_1_0/models/tourspot.py
from datetime import datetime

from mongoengine import *


class TourSpot(document):
    tourId = StringField(required=True)
    userId = StringField(required=True)
    date = DateTimeField(required=True)
    rate = StringField(required=True)
    content = StringField(required=True)

    comment = [{"userId", "date", "rate", "content"}]

File: Node.js/models/tourspot.js
const mongoose = require('mongoose');

const TourSpot = new mongoose.Schema({
  placeid: String,
  comment: [{
    userId: { type: mongoose.Schema.Types.ObjectId, ref: 'user' },
    date: { type: Date, default: Date.now() },
    rate: { type: Number, default: 0 },
    content: String,
  }],
});

module.exports = mongoose.model('tourspot', TourSpot);
